generative/3d_ldm/config/config_train_16g.json

```diff
@@ -0,0 +1,61 @@
+{
+	"channel": 0,
```

Comment:
should we format this JSON file?

Thanks.

Reply:
Sorry. Could you help explain it more? I did not understand it...

Reply:
Sure. Addresed it in another PR https://github.com/Project-MONAI/tutorials/pull/1357